OMF_API/AppSettings.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.Collections.Generic;
+
+namespace OMF_API
+{
+    public class AppSettings
+    {
+        /// <summary>
+        /// The list of endpoints to connect and send to
+        /// </summary>
+        public IList<Endpoint> endpoints { get; set; }
```

Comment:
So I see that you took out the underscores, but we typically use PascalCase also (starts with a capital letter). That's generally the standard in dotnet for properties like this. So this becomes `Endpoints`, and `endpointType` becomes `EndpointType`
It's a bit of a nitpick but it is the standard pattern for dotnet and our dotnet samples.

Reply:
Java has tainted me. We used to only use camelCase and only capitalize class names. I have changed all the properties to PascalCase